#
# Solaris specific patch, which updates proftpd.8 manpage,
# so it contains correct location of various proftpd files.
#

--- a/src/proftpd.8.in
+++ b/src/proftpd.8.in
@@ -109,10 +109,12 @@ This is enabled by default, if the \fB--enable-ipv6\fP configure option is
 used.
 .SH FILES
 .PD 0
-.B @SBINDIR@/proftpd
+.B /usr/lib/inet/proftpd
 .br
 .B @SYSCONFDIR@/proftpd.conf
 .br
+.B @SYSCONFDIR@/ftpusers
+.br
 .B @BINDIR@/ftpwho
 .br
 .B @BINDIR@/ftpcount
